src/psyclone/tests/psyir/transformations/profile_test.py

```diff
@@ -523,7 +521,8 @@ def test_transform(capsys):
     schedule._const_loop_bounds = True
 
     prt = ProfileTrans()
-    assert str(prt) == "Insert a profile start and end call."
+    assert str(prt) == "Create a sub-tree of the PSyIR that has " \
```

Comment:
Now that I get to here, I'm concerned that this isn't a very user-friendly message although it's only the `__str__` method so perhaps that doesn't matter. It's not something your average user will see.

Reply:
I kind of agree, but also note that the previous text to a certain degree could be called wrong: the transformation does NOT insert start and end calls, that is just a 'side effect' of the node that is being inserted. So I think this is certainly an improvement.